.github/workflows/test-sqlite-support.yml

```diff
@@ -0,0 +1,53 @@
+name: Test
```

Comment:
We should change this to `Test SQLite versions` otherwise the GitHub Actions UI has a bunch of things all called `Test`:

![CleanShot 2024-06-13 at 09 42 56@2x](https://github.com/simonw/datasette/assets/9599/f08acc62-292f-486c-9a56-f609f0712471)